CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🗑️ Deprecation
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
+- [7385](https://github.com/vegaprotocol/vega/issues/7385) - Deprecating the `X-Vega-Connection` HTTP header in `datanode` `API` and `REST` and `GraphQL` gateways.
```

Comment:
You removed the headers `Grpc-Metadata-*`. That should appear in a "Removal" section in the CHANGELOG.

Reply:
@ValentinTrinque You mean "Breaking Changes"? There is no "Removal" section in the CHANGELOG

Reply:
Yes this is what I meant